scheduler/main.go

```diff
@@ -0,0 +1,100 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/mailbadger/app/entities"
+	"github.com/mailbadger/app/queue"
+	"github.com/mailbadger/app/storage"
+	"github.com/mailbadger/app/utils"
+)
+
+func main() {
+	driver := os.Getenv("DATABASE_DRIVER")
+	conf := storage.MakeConfigFromEnv(driver)
+	s := storage.New(driver, conf)
+
+	now := time.Now()
+
```

Comment:
It would be nice to add info logs, indicating when a job is starting and is finished, also we may add the time it took for completion.